02-KernelModules_CharacterDevice/concdev.c

```diff
@@ -0,0 +1,182 @@
+#include "concdev.h"
+#include <linux/fs.h>
+#include <linux/types.h>
+#include <linux/kernel.h>
+#include <linux/slab.h>
+#include <linux/cdev.h>
+#include <asm/uaccess.h>
+#include <proc_info.c>
+#include <sysfs_control.c>
+
+#define DEVICE_NAME "concdev"
+
+static int major_number;
+static int minor_number;
+static int count_dev;
+
+static struct cdev *cdev;
+size_t size_buffer;
+size_t capacity;
+
+static char *buffer;
+
+static struct file_operations fops =
+{
+	.owner = THIS_MODULE,
+	.open = cdev_open,
+	.release = cdev_release,
+	.read = cdev_read,
+	.write = cdev_write,
+};
+
+int create_cdevice(int major, int minor, int count, size_t size)
+{
+	int result;
+    dev_t dev = 0;
```

Comment:
```suggestion
         dev_t dev = 0;
```
Please fix last problem in  code style